### PR TITLE
fix: backslash in tagged template

### PR DESCRIPTION
**Pull request by Jack-Works**
This PR fixes `\n` being extracted to `\\n`.

### CODE REVIEW

File: src/ast_utils.rs [hunk 1]
```diff
@@ -52,14 +52,22 @@ pub fn get_expr_as_string(val: &Box<Expr>) -> Option<String> {
...
   }
 }
 
+pub fn get_template_string_cooked_string(element: &TplElement) -> String {
+  if let Some(cooked) = &element.cooked {
+    cooked.to_string()
+  } else {
+    element.raw.to_string()
+  }
+}
```
Comment (timofei-iatsenko) [inline src/ast_utils.rs:69]: Could be done as a one-liner 

```rust
tpl.cooked.as_ref().unwrap_or(&tpl.raw).to_string()
```

Also, let's rename a function to 

`get_tpl_cooked_or_raw_string`

**Review by timofei-iatsenko — CHANGES_REQUESTED** [review]
There two changes in the code, but only one test added. It seems there are cases which is not covered. 

The second case might be

```jsx
<Trans>{`Multiline\nstring`}</Trans>
```
 (please test it with and without fix, because i just took it from mi mind)

**Comment by Jack-Works** [comment]
Hi @timofei-iatsenko thanks for the review, I have updated the PR and added one case, but I cannot fix that case. I tried 2 to 3 different ways but to fix that case it looks like I need to rewrite a lot.

**Comment by Jack-Works** [comment]
looks like the new major version no longer does anything to the strings in tagged templates, closed for now
https://js-lingui-git-next-crowdin.vercel.app/releases/migration-5#no-whitespace-cleaning-in-js-macros

**Pull request closed without merging**